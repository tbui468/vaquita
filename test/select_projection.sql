create database sol;
open sol;

create table planets (id int8 key, name text, mass int8, atmosphere bool);
insert into planets (id, name, mass, atmosphere) values (1, "Neptune", 20, false), (2, "Mars", 10, true), (3, "Venus", 10, false);
select id, name from planets;
drop table planets;

close sol;
drop database sol;
exit;
