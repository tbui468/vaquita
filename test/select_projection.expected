created database sol
opened database sol
created table planets
inserted 4 record(s) into planets
inserted 3 record(s) into planets
Neptune, 
Mars, 
Venus, 
Earth, 
Mercury, 
Jupiter, 
Saturn, 
dropped table planets
closed database sol
dropped database sol
